dask/dataframe/io/parquet/core.py

```diff
@@ -240,11 +240,13 @@ def read_parquet(
         installed, and falls back to ``pyarrow`` otherwise. Note that in the
         future this default ordering for 'auto' will switch, with ``pyarrow``
         being used if it is installed, and falling back to ``fastparquet``.
-    gather_statistics : bool, default None
-        Gather the statistics for each dataset partition. By default,
-        this will only be done if the _metadata file is available. Otherwise,
-        statistics will only be gathered if True, because the footer of
-        every file will be parsed (which is very slow on some systems).
+    calculate_divisions : bool, default False
+        Whether to use Parquet metadata statistics (when available) to
```

Comment:
Just clarifying - "when available" refers to whether any of the individual parquet files have column statistics available for the index columns, not whether the `_metadata` file is present?

Reply:
Yes, there is not requirement for the min/max statistics to be defined for a given column. So, this is just meant to say that we will not calculate divisions if these statistics are missing. I'll try to elaborate a bit more.